lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -214,21 +214,21 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
```

Comment:
Actually @mattdrayer , We are hitting the model dark_lang in navigation.html. Due to that we need to increment `1` in db calls.